### PR TITLE
Add cities, Mar 16 2016

### PR DESCRIPTION
**Pull request by hallaathrad**

### CODE REVIEW

**Comment by alayek** [comment]
@hallaathrad I know you know the lists of these cities better, but I would rather wait for someone to review this before merging. Whom should I assign it to?

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 1]
```diff
@@ -201,6 +204,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
   - [Linyi](https://www.facebook.com/groups/free.code.camp.Linyi/)
   - [Nanjing](https://www.facebook.com/groups/free.code.camp.nanjing/)
   - [Shanghai](https://www.facebook.com/groups/963661723712718/)
```
Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: @hallaathrad why is the Shanghai group named like a number?

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 2]
```diff
@@ -1204,6 +1210,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
     - [Greenville](https://www.facebook.com/groups/free.code.camp.greenville.nc/)
     - [Hickory](https://www.facebook.com/groups/free.code.camp.hickory.nc)
     - [Jacksonville](https://www.facebook.com/groups/free.code.camp.jacksonville.nc/)
+    - [Oriental](https://www.facebook.com/groups/orientalnc/)
```
Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: this is a closed group

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 3]
```diff
@@ -104,6 +104,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
   - [Jacarei](https://www.facebook.com/groups/free.code.camp.sao.paulo.jacarei/)
   - [Joao Pessoa](https://www.facebook.com/groups/free.code.camp.joao.pessoa/)
   - [Joinville](https://www.facebook.com/groups/free.code.camp.joinville/)
+  - [Juiz de Fora](https://www.facebook.com/groups/free.code.camp.juiz.de.fora/)
   - [Londrina](https://www.facebook.com/groups/free.code.camp.londrina/)
   - [Lucas do Rio Verde](https://www.facebook.com/groups/1683221241966841/)
```
Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: This is another such numeric group @hallaathrad

**Comment by abhisekp** [comment]
Here is a complete list of camp sites which are using numeric names
http://hastebin.com/opesejaqoh.md

File: List-of-Free-Code-Camp-city-based-Campsites.md (hunk 2 above)
Comment (hallaathrad) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: this was my mistake. that's the wrong link... to be amended in tonight's commit.

File: List-of-Free-Code-Camp-city-based-Campsites.md (hunk 3 above)
Comment (hallaathrad) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: I'm not sure I understand what you mean as this being a numeric group

**Comment by hallaathrad** [comment]
These are mere URLs, and if you go into the file's history most of these were added even before I came onboard. Currently, this is not causing any issues and very few people actually type the full URL to access their groups. 

However, I do agree with you on that it'd be more "appropriate" and I'll approach this issue after finishing https://github.com/FreeCodeCamp/wiki/issues/304, which as you can see contains quite a few of these numeric URLs. Thanks for the list of these. I've saved it locally to address it in due time.

Meanwhile, the focus of this PR is to daily add new cities and remove the ones reported closed. Not to be blocked by secondary concerns in formatting.

Blocking it until tonight's commit to correct the wrong link for Oriental, NC.

**Comment by hallaathrad** [comment]
Somehow the branch won't let me push. PR from fork https://github.com/FreeCodeCamp/wiki/pull/612

**Pull request closed without merging**